api/USB.json

```diff
@@ -40,6 +40,55 @@
           "deprecated": false
         }
       },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "tags": [
+            "web-features:webusb"
+          ],
```

Comment:
I don't think we need to duplicate the tags from the parent feature.

@ddbeck Can you confirm?

```suggestion
```

Reply:
I see most of this hasn't add a tag, but still some of them added one, like [websocket](https://github.com/skyclouds2001/browser-compat-data/blob/webusb/api/WebSocket.json#L158) [xmlhttprequest](https://github.com/skyclouds2001/browser-compat-data/blob/webusb/api/XMLHttpRequest.json#L102)

Reply:
@caugner Tags do _not_ inherit. But at least for the next couple of months (until we've completed the backfilling in web-features), I wouldn't worry about setting tags on new compat objects. We'll sort them all eventually, then have a better sense of what the guidance should be on BCD.